app/models/user.rb

```diff
@@ -19,6 +19,10 @@ def superadmin?
     return has_role? :superadmin
   end
 
+  def generate_password
+    ((1..9).to_a+('a'..'z').to_a).shuffle[0,10].join
```

Comment:
Are you sure this is the best way to generate random password? Do confirm it once.
Also, a space after comma and before & after any operator (here, +).